crates/core/c8y_smartrest/src/smartrest_serializer.rs

```diff
@@ -39,32 +39,33 @@ impl Default for SmartRestSetSupportedLogType {
     fn default() -> Self {
         Self {
             message_id: "118",
-            supported_operations: vec!["software-management".into()],
+            supported_operations: vec!["software-management"],
         }
     }
 }
 
 impl<'a> SmartRestSerializer<'a> for SmartRestSetSupportedLogType {}
 
 #[derive(Debug, Deserialize, Serialize, PartialEq)]
-pub struct SmartRestSetSupportedOperations {
+pub struct SmartRestSetSupportedOperations<'a> {
     pub message_id: &'static str,
-    pub supported_operations: Vec<&'static str>,
+    pub supported_operations: Vec<&'a str>,
```

Comment:
Why not simply a vector of `String`?

Reply:
I wanted to keep the code fell the same across all the structs in this file where pretty much everywhere are `'static str`